pageserve/app.py

```diff
@@ -15,47 +15,94 @@
 """
 
 import os
-import requests
+from flask import Flask, render_template, request, url_for, session, redirect
+from werkzeug.exceptions import BadRequestKeyError  # WSGI library for Flask
 
-from flask import Flask, render_template, request, Response
+import requests
 
 app = Flask(__name__)  # pylint: disable=invalid-name
 
 
 @app.route('/v1/', methods=['GET'])
 def index():
     """Displays home page with all past posts."""
-    user = get_user()
-    is_auth = has_edit_access(get_user_info(user,
-                                            app.config['USERS_ENDPOINT']
-                                            + "authorization"))
     posts = get_posts()
```

Comment:
just wondering, why assign posts to a temp variable, but then we inlined 'auth='

Reply:
Thanks, not sure why, it was like that when I found it. This and the same things with `events = get_events` fixed in https://github.com/knative-portability/large-events/pull/37/commits/0ff93b6261cef48ed24ed0082ab6793027e6031a